### PR TITLE
Feature/add meta tag settings

### PR DESCRIPTION
**Pull request by HLTVProxy**
- add meta tag settings in Program, ProgramPackage and Blog
- extract SeoSettings and OpenGraphSettings to compoenent

### CODE REVIEW

**Comment by HLTVProxy** [comment]
[(2) 【課程/課程組合/文章】OG標籤、Page title 設定](https://trello.com/c/sBRj82H7/3703-2-%E3%80%90%E8%AA%B2%E7%A8%8B-%E8%AA%B2%E7%A8%8B%E7%B5%84%E5%90%88-%E6%96%87%E7%AB%A0%E3%80%91og%E6%A8%99%E7%B1%A4%E3%80%81page-title-%E8%A8%AD%E5%AE%9A)

File: src/components/form/SeoSettingsBlock.tsx [hunk 1]
```diff
@@ -4,41 +4,50 @@ import { useAuth } from 'lodestar-app-element/src/contexts/AuthContext'
...
-        ...pageAdmin.metaTags,
-        seo: { pageTitle: values.pageTitle, description: values.description, keywords: values.keywords },
+
+    updateMetaTag({
+      variables: {
+        id: id,
+        metaTags: {
+          ...metaTags,
+          seo: { pageTitle: values.pageTitle, description: values.description, keywords: values.keywords },
+        },
```
Comment (sunnylochun) [inline src/components/form/SeoSettingsBlock.tsx:50]: another pattern
use Handling Events onSubmit
passing out event
and update outside
pure component
just mention.... dont need to change

**Review by sunnylochun — APPROVED** [review]

**Comment by sunnylochun** [comment]
data key metaTags 
expect array
but object
TBD

File: src/components/form/SeoSettingsBlock.tsx [hunk 2]
```diff
@@ -4,41 +4,50 @@ import { useAuth } from 'lodestar-app-element/src/contexts/AuthContext'
...
   description: string
   keywords: string
 }
 
-const CraftPageSeoSettingBlock: React.VFC<{
-  pageAdmin: CraftPageAdminProps | null
+const SeoSettingsBlock: React.VFC<{
+  id?: string
+  metaTags?: MetaTag | null
+  updateMetaTag: (options?: any) => Promise<any>
```
[the remark this answers is not in the document]
Comment (HLTVProxy) [reply src/components/form/SeoSettingsBlock.tsx:20]: Because component props type is hard to defined, so I define any

**Review by sunnylochun — APPROVED** [review]

**Pull request merged**